File: app/src/main/java/huce/fit/appreadstories/story/list/download/StoryDownloadView.java
package huce.fit.appreadstories.story.list.download;

import java.util.List;

import huce.fit.appreadstories.model.Story;

public interface StoryDownloadView {

    void setData(List<Story> storyList);
    void openStoryDownload();
    void deleteStoryDownload();
}

File: app/src/main/java/huce/fit/appreadstories/story/download/StoryDownloadView.kt
package huce.fit.appreadstories.story.download

import huce.fit.appreadstories.model.Chapter
import huce.fit.appreadstories.model.Story


interface StoryDownloadView {
    fun setDataStory(story: Story)
    fun setChapterList(chapterList: List<Chapter>)
}
